Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -68,6 +70,49 @@ protected void RenderMRTKLogo()
             MixedRealityEditorUtility.RenderMixedRealityToolkitLogo();
         }
 
+        /// <summary>
+        /// Draws a documentation link for the service.
+        /// </summary>
+        protected void RenderDocLink(Object profileObject)
+        {
+            if (profileObject == null)
+            {   // Can't proceed if profile is null.
+                return;
+            }
+
+            if (!MixedRealityToolkit.IsInitialized || !MixedRealityToolkit.Instance.HasActiveProfile)
+            {   // Can't proceed without an active profile
+                return;
+            }
+
+            // Find the service associated with this profile
+            MixedRealityServiceProfileAttribute profileAttribute = profileObject.GetType().GetCustomAttribute<MixedRealityServiceProfileAttribute>();
+            if (profileAttribute == null)
+            {   // Can't proceed without the profile attribute.
+                return;
+            }
+
+            IMixedRealityService service;
+            if (MixedRealityToolkit.Instance.ActiveSystems.TryGetValue(profileAttribute.ServiceType, out service))
+            {
+                DocLinkAttribute docLink = service.GetType().GetCustomAttribute<DocLinkAttribute>();
+
+                if (docLink != null)
+                {
+                    GUIContent buttonContent = new GUIContent();
```

Comment:
nit: maybe simplify to this

            var buttonContent = new GUIContent()
            {
                image = MixedRealityEditorUtility.HelpIcon,
                text = " Documentation",
                tooltip = docLink.URL,
            };